hpOneView/resources/networking/fc_networks.py

```diff
@@ -29,11 +29,10 @@
 
 standard_library.install_aliases()
 
+from hpOneView.resources.resource import Resource, ResourcePatchMixin
 
-from hpOneView.resources.resource import ResourceClient
 
-
-class FcNetworks(object):
+class FcNetworks(Resource, ResourcePatchMixin):
```

Comment:
In python, class hierarchy is defined from right to left which means ResourcePatchMixin is a base class here extended by Resource. Don't think thats true here.

Reply:
In multiple inheritance class definition, the order of class doesn't mean one is extending from another class. Order matters when it comes to method resolution order that anyways doesn't make an issue here as Mixin classes don't have __init__ or any common methods.

Anyways, thanks for pointing out this. I have to change the order to align with the style I am following in other resource classes.